manager.js

```diff
@@ -0,0 +1,25 @@
+const Employee = require('./EmployeeApp');
+
+class Manager extends Employee {
+    #bonus;
+
+    constructor(id, name, department, salary, bonus) {
+        super(id, name, department, salary);
+        this.#bonus = bonus;
+    }
+
+    getBonus() {
+        return this.#bonus;
+    }
+
+    getSalary() {
+        // Calculate total salary including bonus
+        return super.getSalary() + this.#bonus;
+    }
```

Comment:
You can use getters on this one also, so that someone can do something like this `manager.bonus` or something like `manager.salary`